opm/autodiff/BlackoilWellModel.hpp

```diff
@@ -70,7 +70,9 @@ END_PROPERTIES
 namespace Opm {
 
         /// Class for handling the blackoil well model.
-        template<typename TypeTag>
+        template<typename TypeTag,
+                 typename VFPInjProp=VFPInjProperties,
+                 typename VFPProdProp=VFPProdProperties>
```

Comment:
This looks like something that will ripple through the whole code? Why must this be a template argument?

Reply:
look at the code.